### PR TITLE
Minor format and markup corrections

### PR DESCRIPTION
**Pull request by pnavarrc**
Minor format corrections to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 # express-interceptor
 
->A tiny Express response interceptor.
+_A tiny Express response interceptor_
```
Comment (pnavarrc) [inline README.md]: The `>` prefix is usually used for quotations

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-## Raison d'etre
+## Raison d’être
 
 We tested other packages that offer a similar solution, but found they are difficult to organize, to maintain logic, and to process responses before they are sent. We developed a solution to this problem that offers a declarative-structured way to organize code, and that defines custom middleware for response processing.
 
 Some use cases include:
 
-- Transpile custom elements into standard HTML elements (Sesame).
+- Transpile custom elements into standard HTML elements.
```
Comment (pnavarrc) [inline README.md]: Users of the module will not understand the reference to sesame.

File: README.md [hunk 3]
```diff
@@ -22,11 +22,12 @@ Some use cases include:
 
 ## Usage
 
-* Install the package
+Install the package
+
```
Comment (pnavarrc) [inline README.md]: Removed the list format (only two items). The list also avoids the installation command of being rendered properly.

File: README.md [hunk 4]
```diff
@@ -22,11 +22,12 @@ Some use cases include:
...
 
     npm install --save express-interceptor
 
-* Define your interceptor
+Define your interceptor
```
Comment (pnavarrc) [inline README.md]: Is it possible to include comments in the code snippet? Maybe the interceptor could be defined with a variable name, and add more comments to the returned object saying what it does:

````
var addFooterInterceptor = interceptor(function(req, res) {
  return {
    // Only HTML responses will be intercepted
    isInterceptable: function() {},
    // Appends a paragraph at the end of the response body
    intercept: function(body, send) {}
  };
});
````

File: README.md [hunk 5]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
 
 You're defining an interceptor that will be executed whenever the response contains html as Content-Type. If this is true, it will append a child at the end of the body.
 
-Find other examples at [/examples folder](https://github.com/axiomzen/express-interceptor/tree/master/examples).
+See [more examples](https://github.com/axiomzen/express-interceptor/tree/master/examples).
 
 ## API
 
-* * `isInterceptable()`: is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
+* `isInterceptable()` (required): is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
```
Comment (pnavarrc) [inline README.md]: The `*` were being interpreted as a nested list. Adding (required) is more explicit.

File: README.md [hunk 6]
```diff
@@ -1,20 +1,20 @@
...
+Express-interceptor allows you to define a previous step before sending a response. This allows you to do anything you want with the response, such as processing, transforming, replacing, or logging it. Express-interceptor allows you to avoid calling `next()` over and over. Further more, you can avoid managing nested scopes. Using a declarative API, it’s simple to use and maintain.
 
 [![NPM](https://nodei.co/npm/express-interceptor.png)](https://nodei.co/npm/express-interceptor/)
 
 [![Build Status](https://travis-ci.org/axiomzen/express-interceptor.svg)](https://travis-ci.org/axiomzen/express-interceptor) [![Dependencies](https://david-dm.org/axiomzen/express-interceptor.png)](https://david-dm.org/axiomzen/express-interceptor.png)
 
-## Raison d'etre
+## Raison d’être
 
 We tested other packages that offer a similar solution, but found they are difficult to organize, to maintain logic, and to process responses before they are sent. We developed a solution to this problem that offers a declarative-structured way to organize code, and that defines custom middleware for response processing.
```
Comment (pnavarrc) [inline README.md]: I recommend writing the README in a more objective voice, “packages that offer similar solutions lack of ...”, instead of “we tested other packages...”. Also, in _raison d’être_, is more important to state the problem that the package solve, then the how (architectural differences, patterns) and _maybe_ mention that other packages lack of that (not necessary in my opinion).

File: README.md [hunk 7]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
+* `intercept(body, send)` (required): Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
 
 * `afterSend(oldBody, newBody)`: This method will be called after sending the response to the client – after the `done()` callback in the `send()` method is executed. This method would typically be used to cache something, log stats, fire a job, etc.
 
-(*) Required methods.
 
 ## Similar to
 
 - [express-hijackresponse](https://github.com/papandreou/express-hijackresponse)
-Has issues with cache; code is difficult to maintain.
```
Comment (pnavarrc) [inline README.md]: Is good to mention other packages, but “code is difficult to maintain” is just an opinion. If the package is not very well writen, just skip the mention, or let the user judge the quality of similar modules.

File: README.md [hunk 8]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
+* `isInterceptable()` (required): is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
 
-* * `intercept(body, send)`: Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
+* `intercept(body, send)` (required): Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
 
 * `afterSend(oldBody, newBody)`: This method will be called after sending the response to the client – after the `done()` callback in the `send()` method is executed. This method would typically be used to cache something, log stats, fire a job, etc.
```
Comment (pnavarrc) [inline README.md]: “etc” is not a good way to finish a sentence, “among other things” might sound better.

File: README.md [hunk 9]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
 
-Find other examples at [/examples folder](https://github.com/axiomzen/express-interceptor/tree/master/examples).
+See [more examples](https://github.com/axiomzen/express-interceptor/tree/master/examples).
 
 ## API
 
-* * `isInterceptable()`: is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
+* `isInterceptable()` (required): is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
 
-* * `intercept(body, send)`: Enables you to process the complete response in `body`, as a  properly encoded String. When you're finished with what you wish to do, call `send(newBody)` passing `newBody` as the content you wish to send back to the client.
```
Comment (pnavarrc) [inline README.md]: Process the body as a encoded string (use `String` if it‘s the type, string if it’s an instance). After processing the body, invoke `send`, passing the updated content as argument. The updated content will be sent to the client.

File: README.md [hunk 10]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
 - [express-hijackresponse](https://github.com/papandreou/express-hijackresponse)
-Has issues with cache; code is difficult to maintain. 
+Has issues with cache; code is difficult to maintain.
 
 - [tamper](https://www.npmjs.com/package/tamper)
 Similar functionality but different APIs and internals.
 
 ## Words of advice
 
-This module is new and tests are appreciated. Some edge cases may need fixing.
```
Comment (pnavarrc) [inline README.md]: This sounds like the module is poor quality. The package doesn’t have a contributing guide, so it might be difficult to potential contributor to add new tests.

When warning users about potential issues, add a link to the issues and ask the users to report potential bugs.

File: README.md [hunk 11]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
 
 - [tamper](https://www.npmjs.com/package/tamper)
 Similar functionality but different APIs and internals.
 
 ## Words of advice
 
-This module is new and tests are appreciated. Some edge cases may need fixing. 
+This module is new and tests are appreciated. Some edge cases may need fixing.
 
-Not recommended for intercepting and transforming big responses.
```
Comment (pnavarrc) [inline README.md]: Why? Performance issues? How big? Depends on the transformation or it will be a problem even for small modifications of big responses?

File: README.md [hunk 12]
```diff
@@ -57,31 +58,30 @@ app.listen(3000);
...
 
 ## Words of advice
 
-This module is new and tests are appreciated. Some edge cases may need fixing. 
+This module is new and tests are appreciated. Some edge cases may need fixing.
 
-Not recommended for intercepting and transforming big responses. 
+Not recommended for intercepting and transforming big responses.
 
 Activate debug with `DEBUG=express-interceptor npm test`
```
Comment (pnavarrc) [inline README.md]: This should probably be in a “How to contribute” section.

File: README.md (hunk 4 above)
Comment (p1nox) [inline README.md]: :+1:

File: README.md (hunk 7 above)
Comment (flockonus) [inline README.md]: I'd say in this case it's more than an opinion because they hack really deeply into Node internals, which makes the code brittle to changes. And it's not maintained anymore.
Altho agree we could phase differently, but there must be a reason why not to use the most pop. package in the category

File: README.md (hunk 6 above)
Comment (flockonus) [inline README.md]: Reading it again, line 5 already have very similar motivation, so perhaps it should be either 5 xor 13?

File: README.md [hunk 13]
```diff
@@ -55,35 +61,53 @@ app.listen(3000);
...
+
+```html
+<html>
+<head></head>
+<body>
+  <p>"Hello world"</p>
+  <p>From interceptor!</p>
+</body>
+</html>
+```
```
Comment (flockonus) [inline README.md]: Very nice

**Comment by flockonus** [comment]
Great improvements! I'd added/changes some stuff in the commit, feel free to improve it further

- removed my French'ism fun, along with a whole paragraph that wasn't adding much
- put the badges up, but should it be at the top?
- come on, devs should know _npm i_, if they don't and can learn it here (it's a perk!)
- applied Pablo's comment on API, changes a lot last lines, might need revision

**Comment by pnavarrc** [comment]
I think is good to leave the `npm install <package-name>` line, most developers will copy-paste that into the console.

> put the badges up, but should it be at the top?
I think so. The zenhub badge color is too similar to the warning/not passing color, I think we should change that at some point.

**Comment by p1nox** [comment]
Ok, so are we good to merge?

**Comment by flockonus** [comment]
Yes, `npm i` == `npm install` :grin:

**Comment by pnavarrc** [comment]
Oh, but in this context `install` might be better, for people that don’t know the `i` shortcut (which is better to type, I agree)

**Comment by flockonus** [comment]
I'm hoping they will most likely either know or learn by association like i did once, the --save is a bit arguable tho, but it would a good leverage in spreading the package

File: README.md (hunk 7 above)
Comment (dgipps) [inline README.md]: Instead of saying "code is difficult to maintain" say "code is no longer maintained and it relies too much on node internals for us to want to fork it and maintain it"

Comment (pnavarrc) [inline README.md]: :+1:

**Comment by pnavarrc** [comment]
:beers:
